plugins/processors/template/README.md

```diff
@@ -59,4 +59,32 @@ Add the year as a tag, similar to the date processor:
   template = '{{.Time.UTC.Year}}'
 ```
 
+Add all fields as a tag:
+
+```toml
+[[processors.template]]
+  tag = "message"
+  template = 'Fields: {{ .FieldList }}'
+  # Advanced example:
+  # template 'Fields: {{ range $name, $value := .FieldList }}{{$name}}:{{$value}} {{end}}'
```

Comment:
I would prefer you remove the simple example and instead uncomment and show the more advanced example here. I think that is more likely what user is going to want versus the "map[.]" text.